ibis/backends/sqlite/__init__.py

```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 
 
-from ibis.sql.sqlite.client import SQLiteClient
-from ibis.sql.sqlite.compiler import dialect, rewrites  # noqa: F401
+from ibis.backends.sqlite.client import SQLiteClient
```

Comment:
Those are the imports that would be great to be relative:
```suggestion
from .client import SQLiteClient
```

Same for the others too.